docs/Using Price Feeds/using-chainlink-reference-contracts.md

```diff
@@ -2,34 +2,40 @@
 layout: nodes.liquid
 section: ethereum
 date: Last Modified
-title: "Introduction to Data Feeds"
+title: "Chainlink Data Feeds Documentation"
 permalink: "docs/using-chainlink-reference-contracts/"
-whatsnext: {"Get the Latest Price":"/docs/get-the-latest-price/", "API Reference":"/docs/price-feeds-api-reference/", "Contract Addresses":"/docs/reference-contracts/"}
+whatsnext: {"Get the latest price of a Data Feed":"/docs/get-the-latest-price/", "Learn more about API references for using Data Feeds":"/docs/price-feeds-api-reference/", "Retrieve contract addresses to use Data Feeds":"/docs/reference-contracts/"}
 metadata:
   title: "Introduction to Data Feeds"
   description: "Add data to your smart contracts and applications. Chainlink data feeds include BTC/USD, BTC/ETH, ETH/USD and more!"
 ---
 ![Chainlink Abstract Banner](/files/2306b8b-Decentralized_Oracles_V3.png)
 
-## Connect your contracts to the outside world
-
 Chainlink Data Feeds are the quickest way to connect your smart contracts to the real-world data such as asset prices. One use for data feeds is to retrieve the latest pricing data of an asset in a single call and use that data either on-chain in a smart contract or off-chain in another application of your choice.
 
 If you already have a project started and would like to integrate Chainlink, you can [add Chainlink to your existing project](../create-a-chainlinked-project/#install-into-existing-projects) by using the [`chainlink` NPM package](https://www.npmjs.com/package/@chainlink/contracts).
 
 See the [Data Feeds Contract Addresses](/docs/reference-contracts/) page for a list of networks and proxy addresses.  
 
-For important updates regarding the use of Chainlink Price Feeds, users should join the official Chainlink Discord and subscribe to the data-feeds-user-notifications channel: https://discord.gg/Dqy5N9UbsR
+**Topics**
```

Comment:
I like this change. Shall we switch all of our `**Table of contents**` entries to say `**Topics**` instead?

Reply:
I think if we agree to change it on this page, we should carry it out to all pages for consistency.

Reply:
Resolved